tests/remote/test_against_real_server.py

```diff
@@ -34,7 +34,9 @@ def test_happy_path(base_url: str, api_key: str) -> None:
         api_key=api_key,
         url=base_url,
     )
-    backend = provider.get_backend('EMU:1Q:LESCANNE_2020')
-    job = execute(c, backend)
+    backend = provider.get_backend(
+        'EMU:1Q:LESCANNE_2020', average_nb_photons=6.0
+    )
+    job = execute(c, backend, shots=100)
```

Comment:
About testing, I also did it on a dedicated python file and checked the job was created and ran well.